delete-vm.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# vm should not exist
+multipass delete --purge vtvl && echo "deleted vtvl"
+
+# ssh key should not exist
+rm id_ed25519*
```

Comment:
You should add any ssh keys to the .gitignore to make sure they never get added to git.

Reply:
Amazing idea! You're a genius! Done and tested!